packages/manager/modules/pci/src/projects/project/failover-ips/failover-ips.module.js

```diff
@@ -23,6 +24,7 @@ angular
   ])
   .config(routing)
   .component('pciProjectFailoverIps', component)
+  .service('PciProjectsProjectInstanceService', instanceService)
```

Comment:
Already defined https://github.com/ovh/manager/blob/master/packages/manager/modules/pci/src/projects/project/instances/instances.module.js#L55

Reply:
This is fixed by adding the module as a dependency. Thanks :)